compiler/modules/comm.diagnostic/diagnostic.bal

```diff
@@ -27,6 +27,7 @@ public type File readonly & object {
     public function directory() returns string?;
     public function lineColumn(Position pos) returns LineColumn;
     public function lineContent(Position|Range range) returns [string, string, string];
+    public function qNameEndPos(Position startPos) returns Position;
```

Comment:
Probably better for this to be qNameRange and return a Range.

Reply:
fixed